cinema/serializers.py

```diff
@@ -59,6 +85,7 @@ class MovieSessionListSerializer(MovieSessionSerializer):
     cinema_hall_capacity = serializers.IntegerField(
         source="cinema_hall.capacity", read_only=True
     )
+    tickets_available = serializers.IntegerField(read_only=True)
```

Comment:
The 'tickets_available' field in 'MovieSessionListSerializer' is defined as read-only, but there is no logic provided to calculate or set this field. Consider implementing a method to calculate available tickets or remove this field if not needed.